src/appleseed/foundation/meta/tests/test_microfacet.cpp

```diff
@@ -114,10 +169,10 @@ TEST_SUITE(Foundation_Math_Microfacet)
 
     template <typename MDF>
     void weak_white_furnace_test(
+        const MDF&                  mdf,
```

Comment:
What's the point of passing `mdf` here since everything is static anyway?

Reply:
The StdMDFAdaptor class defined at the begining of the file.

Reply:
Ok, makes sense, thanks.